modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/value/ValueProviderMediator.java

```diff
@@ -110,9 +108,9 @@ public ValueProviderMediator(T containerModel, Supplier<MuleContext> muleContext
    * @return a {@link Set} of {@link Value} correspondent to the given parameter
    * @throws ValueResolvingException if an error occurs resolving {@link Value values}
    */
-  public Set<Value> getValues(String parameterName, ParameterValueResolver parameterValueResolver,
-                              Supplier<Object> connectionSupplier, Supplier<Object> configurationSupplier,
-                              ConnectionProvider connectionProvider)
+  public Set<org.mule.sdk.api.values.Value> getValues(String parameterName, ParameterValueResolver parameterValueResolver,
```

Comment:
Would this bring problems for tooling? I think they use this class directly. @gsfernandes

Reply:
yep. This ties back into the prior comment of deprecation/replacement being incomplete. This breaks backwards compatibility